lib/cli/src/helpers.ts

```diff
@@ -29,7 +29,13 @@ export function readFileAsJson(jsonPath: string, allowComments?: boolean) {
 
   const fileContent = fs.readFileSync(filePath, 'utf8');
   const jsonContent = allowComments ? stripJsonComments(fileContent) : fileContent;
-  return JSON.parse(jsonContent);
+
+  try {
+    return JSON.parse(jsonContent);
+  } catch(e) {
+    throw new Error('Invalid json on file: ' + filePath);
```

Comment:
Maybe we should log the filename and still throw the original error? I would hate to lose the stack trace for debugging

Reply:
Changed as you suggested. Just added `chalk.red` on it:
```ts
try {
    return JSON.parse(jsonContent);
  } catch(e) {
    console.error(chalk.red('Invalid json on file: ' + filePath));
    throw e;
}
```